hy/macros.py

```diff
@@ -123,13 +129,42 @@ def _import(module, module_name=module_name):
         _import(module)
 
 
-def make_emtpy_fn_copy(fn):
-    argspec = getargspec(fn)
-    formatted_args = formatargspec(*argspec)
-    fn_str = 'lambda {}: None'.format(
-        formatted_args.lstrip('(').rstrip(')'))
-
-    empty_fn = eval(fn_str)
+def make_empty_fn_copy(fn):
+    try:
+        argspec = getargspec(fn)
+    except:
+        return lambda *_: None
```

Comment:
Depending on how we resolve #959 this might have to be `return lambda *arg, **kwarg: None`. But why is `getargspec` failing in the first place? Can we fix that instead?